appserver/jms/jms-core/src/main/java/com/sun/enterprise/connectors/jms/system/ActiveJmsResourceAdapter.java

```diff
@@ -1043,9 +1044,13 @@ private void setLifecycleProperties() throws
             if (tmpString == null) {
                 tmpString = "";
             }
-
             String brokerArgs = tmpString;
 
+            //condition to evaluate the JDK and if version greater than 8 then omit the jre folder
+            //by adding the start-args for the broker configuration
+            if (availableJDKForStartArgs()) {
+                brokerArgs = buildStartArgsForJREHome(java_home);
```

Comment:
This overrides any defined `start_args` value.  We should be more selective and only add (when not already defined) the `-jrehome` argument

Reply:
already attended